src/test/java/net/snowflake/ingest/connection/TestKeyRenewal.java

```diff
@@ -30,7 +30,8 @@ public void doesRegenerateToken()
     KeyPair keyPair = keyGen.generateKeyPair();
 
     // create the security manager;
-    SecurityManager manager = new SecurityManager("account", "user", keyPair, 3, TimeUnit.SECONDS);
+    SecurityManager manager =
+        new SecurityManager("account", "user", keyPair, 3, TimeUnit.SECONDS, null);
```

Comment:
Strange that this test passing since we are passing null and it will not generate new token?